skbeam/core/roi.py

```diff
@@ -430,18 +476,27 @@ def circular_average(image, calibrated_center, threshold=0, nx=100,
     ring_averages : array
         Radial average of the image. shape is (nx, ).
     """
-    radial_val = utils.radial_grid(calibrated_center, image.shape, pixel_size)
-
-    bin_edges, sums, counts = utils.bin_1D(np.ravel(radial_val),
-                                           np.ravel(image), nx,
+    radial_val = utils.radial_grid(calibrated_center, image.shape, pixel_size) 
```

Comment:
Yes, I mean out of the plane. But why should we be building code we need to "trick"? It might be better to take the array which translates pixel position to radial position in as a parameter. This way no tricking is needed.